src/app/components_ngrx/work-item-link/work-item-link.component.html

```diff
@@ -127,17 +127,13 @@
           <ul class="padding-left-0">
             <li *ngFor="let wi of linkDict.workItems; let i = index"
               class="f8-link__list-item">
-              <a [routerLink]="'detail/' + wi.number" *ngIf="detailContext === 'preview'">
+              <a [routerLink]="detailContext === 'preview' ? 'detail/' + wi.number : createRouterLink(wi.number, detailContext)"
+                (click)="detailContext === 'detail' && onLinkClicked(wi.number)">
```

Comment:
@vikram-raj here also `onLinkClicked` function could have been avoided by simply emitting the event and passing the wi number when clicked on the link from the detail page. Also, the check on `click` event is unnecessary in the case when the user is clicking on the workitem link from quick preview.